x/mongo/driver/operation.go

```diff
@@ -1098,78 +1080,6 @@ func (op Operation) addBatchArray(dst []byte) []byte {
 	return dst
 }
 
-func (op Operation) createQueryWireMessage(maxTimeMS uint64, dst []byte, desc description.SelectedServer) ([]byte, startedInformation, error) {
-	var info startedInformation
-	flags := op.secondaryOK(desc)
-	var wmindex int32
-	info.requestID = wiremessage.NextRequestID()
-	wmindex, dst = wiremessage.AppendHeaderStart(dst, info.requestID, 0, wiremessage.OpQuery)
```

Comment:
Should we remove the `wiremessage.OpQuery` constant?

Also, `wiremessage.OpReply` is used in a few places, which I think is only returned when sending `OP_QUERY` (see [OP_REPLY docs](https://www.mongodb.com/docs/manual/legacy-opcodes/#op_reply)).